src/WCSimConstructRealisticPlacement.cc

```diff
@@ -0,0 +1,1168 @@
+#endif
+
+struct RealisticPlacementConfiguration {
+
+    G4ThreeVector EntireDetectOffset; //< Relative offset for
+
+    G4Material* InnerDetectorMaterial;
+    G4Material* OuterDetectorMaterial;
+
+    G4VisAttributes* BlackTyvekVis;
+    G4Material* BlackTyvekMaterial;
+    G4double BlackTyvekInnerRadius;
+    G4double BlackTyvekOuterRadius;
+    G4double BlackTyvekBarrelLength;
+
+    G4VisAttributes* DeadSpaceVis;
+    G4Material* DeadSpaceMaterial;
+    G4double DeadSpaceInnerRadius;
+    G4double DeadSpaceOuterRadius;
+    G4double DeadSpaceBarrelLength;
+
+    G4VisAttributes* OuterDetectorVis;
+    G4double OuterDetectorInnerRadius;
+    G4double OuterDetectorOuterRadius;
+    G4double OuterDetectorBarrelLength;
+
+    G4VisAttributes* InnerDetectorVis;
+    G4double InnerDetectorInnerRadius;
+    G4double InnerDetectorOuterRadius;
+    G4double InnerDetectorBarrelLength;
+
+    G4VisAttributes* WhiteTyvekVis;
+    G4Material* WhiteTyvekMaterial;
+    G4double WhiteTyvekInnerRadius;
+    G4double WhiteTyvekOuterRadius;
+    G4double WhiteTyvekBarrelLength;
+
+    G4VisAttributes* WallTyvekVis;
+    G4Material* WallTyvekMaterial;
+    G4double WallTyvekInnerRadius;
+    G4double WallTyvekOuterRadius;
+    G4double WallTyvekBarrelLength;
+
+    G4VisAttributes* MainWaterTankVis;
+    G4Material* MainWaterTankMaterial;
+    G4double MainWaterTankRadius;
+    G4double MainWaterTankLength;
+
+    G4VisAttributes* RockShellVis;
+    G4Material* RockShellMaterial;
+    G4double RockShellRadius;
+    G4double RockShellLength;
+
+    G4double NFrameCellsPerRow;
+    G4double NRowsPerMainBlock;
+    G4double NRowsPerBottomBlock;
+    G4double NBlocksPerRing;
+
+    G4double RowSeperation;
+    int NSpacesInBlock;
+    int NBlocksUp;
+    int NBlocksAround;
+    G4double CellArcLength;
+
+
+    void Print(){
+      std::cout << "-----------------------------------" << std::endl;
+      std::cout << "HK FD Detector Configuration Table " << std::endl;
+      std::cout << "-----------------------------------" << std::endl;
+      std::cout << "BlackTyvekInnerRadius = " << BlackTyvekInnerRadius << std::endl;
+      std::cout << "BlackTyvekOuterRadius = " << BlackTyvekOuterRadius << std::endl;
+      std::cout << "DeadSpaceInnerRadius = " << DeadSpaceInnerRadius << std::endl;
+      std::cout << "DeadSpaceOuterRadius = " << DeadSpaceOuterRadius << std::endl;
+      std::cout << "WhiteTyvekInnerRadius = " << WhiteTyvekInnerRadius << std::endl;
+      std::cout << "WallTyvekOuterRadius = " << WallTyvekOuterRadius << std::endl;
+      std::cout << "WallTyvekInnerRadius = " << WallTyvekInnerRadius << std::endl;
+      std::cout << "WallTyvekOuterRadius = " << WallTyvekOuterRadius << std::endl;
+      std::cout << "MainWaterTankRadius = " << MainWaterTankRadius << std::endl;
+      std::cout << "MainWaterTankLength = " << MainWaterTankLength << std::endl;
+      std::cout << "RockShellRadius = " << RockShellLength << std::endl;
```

Comment:
```suggestion
      std::cout << "RockShellRadius = " << RockShellRadius << std::endl;
```
Fixing copy-paste error